test/test_fullscheduler_ortoolkit.py

```diff
@@ -1,109 +1,44 @@
 #!/usr/bin/env python
```

Comment:
Is this only testing cbc? If so, this file might be better named `test_fullscheduler_cbc.py`, in line with the names of the other test files that test gurobi and glpk.

Reply:
yup